packages/atomic/playwright-utils/base-page-object.ts

```diff
@@ -0,0 +1,43 @@
+import type {Page} from '@playwright/test';
+import {buildArgsParam} from '@storybook/router';
+import {JSX} from '../dist/types/components';
```

Comment:
I'm still uncertain we'll be able to uses this everywhere :tm: and that the JSX will map always to what's available in Storybook, but we'll see!

Reply:
I tried to use `custom-elements.json` thingy. 

Main problem is that it's a (weakly typed) JSON, and so we end up with way more complex Typescript in this file to just try to have have some type/interface to represent "an actual Atomic web component available with its tag name and its prop". The end goal is just to get intellisense completion when writing a new test.

And for now in test, we don't need to configure slots, JSDoc, events, shadow parts, etc. Nor did we need to with the older Cypress tests: just components properties. So it just seems like over engineering.

Furthermore: Nothing stop us from writing a `page.goto(....)` manually if absolutely necessary.

Reply:
> And for now in test, we don't need to configure slots, JSDoc, events, shadow parts, etc. Nor did we need to with the older Cypress tests: just components properties. So it just seems like over engineering.

I'm pretty sure the slot will be required. It's been used in the Cypress test using the `appendChild` (i.e., look for `appendChild` in `cypress`; most references are "slot" in disguise).

However, most slots could/should be tested by adding a new story.
We can cross that road if/when we reach it.